### DISCUSSION TITLE
jscodeshift @wdio/codemod dependencies required after update?

### DISCUSSION THREAD

**Discussion by zachlysobey** [webdriverio/codemod#57] [Migration Issue] [answered]
The ["From v6 to v7" documentation](https://webdriver.io/docs/v7-migration/) says to install `jscodeshift` and `@wdio/codemod`

> `$ npm install jscodeshift @wdio/codemod`

This command will of course add these to `package*.json` and `node_modules`.

But are these really necessary to linger around after migration? It adds dependency bloat, etc...=/

Ideally, I think we'd want steps to do an upgrade and have the end-product be indistinguishable from a freshly created one.

I also left a comment on WarleyGabriel/demo-webdriverio-cucumber#11, but thought maybe better to include here in *discussions* 🤷

**Comment by christian-bromann** [comment, marked as answer]
No these dependencies are not needed after your migration is done. You don't have to add them to your `package.json`.
